static/amp-app-banner-manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "prefer_related_applications": true,
+  "related_applications": [
+    {
+      "platform": "play",
+      "id": "com.medium.reader",
+      "url": "android-app://com.medium.reader/https/medium.com/p/cb7f223fad86"
```

Comment:
Is there supposed to be a link to a specific article here? I'm assuming the manifest will be shared across a lot of articles, so it seems suspicious that a specific article is embedded into the manifest. (On iOS the equivalent deep-link seems to be per-page, in `<meta name="apple-itunes-app">`.)